### PR TITLE
Add privilege for accessing global sms gateway

### PR DESCRIPTION
**Pull request by AmitPhulera**
## Product Description

https://dimagi-dev.atlassian.net/browse/SAAS-13649
## Technical Summary

This is a followup of https://github.com/dimagi/commcare-hq/pull/31750 where we are restricting Global SMS Gateways page to the user who have `GLOBAL_SMS_GATEWAY`
## Safety Assurance
Has been tested on local and would be putting it on staging to test it.
### QA Plan

NA

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by biyeun — CHANGES_REQUESTED** [review]
Hey @AmitPhulera looking good! I just added some comments to clarify the difference between using the privileges for plans vs superusers

File: corehq/privileges.py [hunk 1]
```diff
@@ -134,6 +136,7 @@
     RELEASE_MANAGEMENT,
     LITE_RELEASE_MANAGEMENT,
     LOADTEST_USERS,
+    GLOBAL_SMS_GATEWAY,
```
[the remark this answers is not in the document]
Comment (AmitPhulera) [reply corehq/privileges.py]: Done.

File: corehq/privileges.py [hunk 2]
```diff
@@ -6,6 +6,8 @@
 CLOUDCARE = 'cloudcare'
 GEOCODER = 'geocoder'
 
+GLOBAL_SMS_GATEWAY = 'global_sms_gateway'
```
[the remark this answers is not in the document]
Comment (AmitPhulera) [reply corehq/privileges.py]: Added at the end with the relevant comment. https://github.com/dimagi/commcare-hq/pull/32049/commits/87d2e30390c44940d47d940aab3bc8703a839730

**Review by calellowitz — APPROVED** [review]
Pending biyeuns approval (and minor language changes) looks good to me

**Review by RynhardPietersen — APPROVED** [review]
Looking good!

**Review by biyeun — APPROVED** [review]
🎉

**Pull request merged**